app/controllers/prioritized_issues_controller.rb

```diff
@@ -44,12 +44,17 @@ def move_to_bucket
   end
 
   def sync
-    prioritized_issue_service.
-      for_prioritized_issue_by_id(params[:prioritized_issue_id]).
-      update_issue_with_values_from_github
-
-    render :partial => "buckets/issue",
-           :locals => {:issue => prioritized_issue_service.prioritized_issue}
+    prioritized_issue_service.for_prioritized_issue_by_id(params[:prioritized_issue_id])
+    before_sync_updated_at = prioritized_issue_service.issue_updated_at
+    prioritized_issue_service.update_issue_with_values_from_github
+    after_sync_updated_at = prioritized_issue_service.issue_updated_at
```

Comment:
While updating this to not re-render the issue if it hasn't been updated I'm starting to really dislike my prioritized_issue_service API. This will probably need redesigned in the future.